base/docs/basedocs.jl

```diff
@@ -789,6 +789,42 @@ run(pipeline(`cat test.txt`, devnull))
 """
 devnull
 
+""" 
+    `\$`
```

Comment:
The backticks are not necessary here. And I would add a space before the next line to separate this visually (as is done for the other keywords).